docs/users/cli-usage.md

```diff
@@ -135,10 +136,12 @@ Get started with Drupal 8 projects on ddev either using a new or existing compos
 ```
 mkdir my-drupal8-site
 cd my-drupal8-site
-ddev config --project-type php
-ddev composer create drupal-composer/drupal-project:8.x-dev --prefer-dist
-ddev config --project-type drupal8
-ddev restart
+ddev config --project-type=drupal8 --docroot=web --create-docroot=true
+ddev start
```

Comment:
@rfay – Randy I'm just trying it out right now and I'm wondering why it's two times `ddev start`? One here and one in the last step.